components/brave_rewards/common/mojom/ledger.mojom

```diff
@@ -207,9 +207,6 @@ enum Result {
   GRANT_ALREADY_CLAIMED = 18,
 
   CONTRIBUTION_AMOUNT_TOO_LOW = 19,
-  VERIFIED_PUBLISHER = 20,
```

Comment:
Would you mind removing `ledger.mojom.PendingContribution` and `ledger.mojom.PendingContributionInfo`, too? Only references are in tests. You can use the code I provided [here](https://github.com/brave/brave-core/pull/17647#discussion_r1164653916).

Reply:
Done.